pkg/imageset/updateinstaller.go

```diff
@@ -179,22 +181,28 @@ func (o *UpdateInstallerImageOptions) Run() error {
 
 	if success {
 		o.log.Debugf("the oc release info command was successful")
-		installerImageFileName := path.Join(o.WorkDir, "installer-image.txt")
-		installerImageBytes, err := ioutil.ReadFile(installerImageFileName)
-		if err != nil {
-			o.log.WithError(err).Error("could not read install image file")
-			return err
+		var installerImage, cliImage string
+		if len(o.InstallerImage) > 0 {
+			o.log.Debugf("installer image override provided - skip installer image resolution")
```

Comment:
This is not actually skipping the installer image resolution. The resolution is being done in the other container of the imageset pod.